doc/_themes/ceph/static/nature.css_t

```diff
@@ -106,92 +106,99 @@ div.sphinxsidebar h4 {
 div.sphinxsidebar h4{
     font-size: 1.1em;
 }
- 
+
 div.sphinxsidebar h3 a {
     color: #e6e8e8;
 }
- 
- 
+
+div.sphinxsidebar h4 a {
+    color: #e6e8e8;
+}
+
 div.sphinxsidebar p {
     color: #888;
     padding: 5px 20px;
 }
- 
+
 div.sphinxsidebar p.topless {
 }
- 
+
 div.sphinxsidebar ul {
-    margin: 10px 5px 10px 20px;
+    margin: 10px 20px;
     padding: 0;
     color: #000;
 }
- 
+
 div.sphinxsidebar a {
     color: #444;
 }
- 
+
 div.sphinxsidebar input {
     border: 1px solid #ccc;
     font-family: sans-serif;
     font-size: 1em;
 }
 
-div.sphinxsidebar input[type=text]{
+div.sphinxsidebar input[type=text] {
     margin-left: 20px;
+    margin-right: 20px;
 }
- 
+
+div.sphinxsidebar input[type=submit] {
+      border-left: 1px solid #ccc !important;
+          margin-left: 20px;
+}
+
 /* -- body styles ----------------------------------------------------------- */
- 
+
 a {
     color: #F05C56;
     text-decoration: none;
 }
 
 a:hover {
-    color: #F05C56;
+    color: #E32E00;
     text-decoration: underline;
 }
- 
+
 div.body h1,
 div.body h2,
 div.body h3,
 div.body h4,
 div.body h5,
 div.body h6 {
+    background-color: #99DAE3;
+    font-weight: normal;
+    color: #212224;
     margin: 30px 0px 10px 0px;
-    padding: 5px 0 5px 0px;
+    padding: 5px 0 5px 10px;
+    // text-shadow: 0px 1px 0 white;
```

Comment:
probably just change "write" to "gray"?

Reply:
@tchaikov Can't agree with this one I'm afraid as that corrupts the font. The case at the moment is that the white shadow is not seen on the white background and is therefore invisible or effectively absent and thus you only see the actual font as it is meant to be displayed.